gradle-sls-packaging/src/main/java/com/palantir/gradle/dist/service/JavaServiceDistributionPlugin.java

```diff
@@ -306,6 +306,11 @@ public void execute(Task _task) {
 
         project.afterEvaluate(_p -> launchConfigTask.configure(task -> {
             task.getJavaAgents().setFrom(javaAgentConfiguration);
+            task.getFullClasspath()
+                    .from(jarTask.get()
+                            .getOutputs()
+                            .getFiles()
+                            .plus(distributionExtension.getProductDependenciesConfig()));
```

Comment:
minor nit: This duplicates the ternary else component on 318-322, since we're always building that provider now, we might as well build a single provider and pass it to both places. I think we always want the value to be identical.

Reply:
Cool, wasn't sure if this was expensive or not...easy to fix.